examples/gallery/maps/tilemaps.py

```diff
@@ -0,0 +1,36 @@
+"""
+Tilemaps
+--------
+The :meth:`pygmt.Figure.tilemap` method allows to plot
+tiles from a tile server or local file as a basemap or overlay.
+"""
+import contextily
+import pygmt
+
+fig = pygmt.Figure()
+fig.tilemap(
+    region=[-157.84, -157.8, 21.255, 21.285],
+    projection="M12c",
+    zoom=14,
```

Comment:
I feel it would be helpful to add some explanation of the `zoom` parameter either within the code or above. Feel free to adjust or shorten the formulation :slightly_smiling_face:.
```suggestion
    # Set level of details (0-22)
    # Higher levels mean a zoom level closer to the Earth's
    # surface with more tiles covering a smaller
    # geographic area and thus more details and vice versa
    # Please note, not all zoom levels are always available
    zoom=14,
```